src/lib/coerce.js

```diff
@@ -521,3 +539,48 @@ function validate(value, opts) {
     return out !== failed;
 }
 exports.validate = validate;
+
+var dtypeStringToTypedarrayType = {
```

Comment:
I think that all of them, except for `Uint8ClampedArray`:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays#Typed_array_views

Might as well add it here for completeness.

Reply:
Added in 5030d3a